cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,20 +1,38 @@
+import {MenuContentPage} from "../page/menu-content.page";
+import {ProductListPage} from "../page/products-list.page";
+import {ShoppingCartPage} from "../page/shopping-cart.page";
+import {LoginPage} from "../page/login.page";
+import {AddressStepPage} from "../page/address-step.page";
+import {ShippingStepPage} from "../page/shipping-step.page";
+import {PaymentStepPage} from "../page/payment-step.page";
```

Comment:
Try to import this pages from the index module

Reply:
Something like this? 
import {MenuContentPage} from "../page/index";

Reply:
That's right 😄